test/client_test.dart

```diff
@@ -25,4 +29,67 @@ void main() {
     expect(session.accessToken,
         'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJhdXRoZW50aWNhdGVkIiwiZXhwIjoxNjExODk1MzExLCJzdWIiOiI0Njg3YjkzNi02ZDE5LTRkNmUtOGIyYi1kYmU0N2I1ZjYzOWMiLCJlbWFpbCI6InRlc3Q5QGdtYWlsLmNvbSIsImFwcF9tZXRhZGF0YSI6eyJwcm92aWRlciI6ImVtYWlsIn0sInVzZXJfbWV0YWRhdGEiOm51bGwsInJvbGUiOiJhdXRoZW50aWNhdGVkIn0.GyIokEvKGp0M8PYU8IiIpvzeTAXspoCtR5aj-jCnWys');
   });
+
+  test('signUp()', () async {
+    final response = await client.signUp(email, password);
+    final data = response.data;
+    final error = response.error;
+    expect(error, isNull);
+    expect(data.accessToken is String, true);
+    expect(data.refreshToken is String, true);
+    expect(data.user.id is String, true);
+  });
```

Comment:
@phamhieu I am not sure if this is sufficient enough to test the validity of data or not. Please let me know what you think.

Reply:
you can check if data.user.email == email address you pass to the method. We don't want it return another user data...